.travis.yml

```diff
@@ -7,6 +7,7 @@ before_install:
 install:
   - ./travis-tool.sh r_binary_install knitr
   - ./travis-tool.sh r_binary_install testthat
+  - ./travis-tool.sh r_install pryr
```

Comment:
I can't see the functionality we need in lazyeval. Maybe if we "convert" the promise to lazy, but that seems a bit convoluted to be. We need only two things: (1) decide if something is a promise, and (2) decide if it was evaluated already. Ideally these should be in base R.